scipy/stats/_stats_py.py

```diff
@@ -4061,7 +4061,7 @@ class AlexanderGovernResult:
     pvalue: float
 
 
-@xp_capabilities()
+@xp_capabilities(jax_jit=False)
```

Comment:
For all these cases where `jax_jit=False` had to be added to `xp_capabilities`, did these functions originally work with the `jit` before this PR, or is it just that `jax_jit=False` was never added because all relevant tests were skipped?